backend/fleet_management/serializers.py

```diff
@@ -82,18 +81,6 @@ def create(self, validated_data):
             drive.passengers.set(passengers)
             drive.save()
 
-            for passenger in passengers:
```

Comment:
This code doesn't look like it's not being used. Also, flake8 did not report it as not needed when I run it on `develop`.
```
(venv) ~/self/pah-fm/backend(develop) $ flake8
./fleet_management/factories.py:6:1: F811 redefinition of unused 'random' from line 1
./fleet_management/signals.py:2:1: F401 'django.core.mail.EmailMessage' imported but unused
./fleet_management/signals.py:2:1: F401 'django.core.mail.get_connection' imported but unused
./fleet_management/signals.py:5:1: F401 'pah_fm.settings.EMAIL_ADDRESS' imported but unused
./fleet_management/signals.py:14:5: F841 local variable 'template' is assigned to but never used
./fleet_management/signals.py:15:5: F841 local variable 'subject' is assigned to but never used
./fleet_management/signals.py:17:5: F841 local variable 'drive' is assigned to but never used
./fleet_management/signals.py:18:5: F841 local variable 'driver' is assigned to but never used
./fleet_management/signals.py:19:5: F841 local variable 'token' is assigned to but never used
./fleet_management/signals.py:21:5: F841 local variable 'passenger' is assigned to but never used
```

Reply:
Discussed this offline with @magul, seems like it is in fact not needed, so I'm giving a green light on this.